pil-analyzer/tests/condenser.rs

```diff
@@ -65,7 +65,7 @@ fn new_witness_column_name_clash() {
 #[test]
 fn create_constraints() {
     let input = r#"namespace N(16);
-    let force_bool: expr -> Constr = |c| c * (1 - c) = 0;
+    let force_bool: expr -> () = |c| c * (1 - c) = 0;
```

Comment:
This should stay as it is, it returns the constraint and does not add it to the global set.